sources/packages/forms/src/form-definitions/studentrequestchange.json

```diff
@@ -444,6 +444,10 @@
           "dataSrc": "values",
           "data": {
             "values": [
+              {
+                "value": "studentdependantsappealparttime",
```

Comment:
Can you please remove parttime from the value and also the other form??

![image](https://github.com/bcgov/SIMS/assets/54600590/8656d1e5-544a-406e-afd7-47472e8f8b78)